html/lesson6/tutorial.md

```diff
@@ -143,3 +143,23 @@ This ends our sixth lesson. How did you find learning HTML and CSS? If you have
 ## Extras
 
 Now that you are familiar with HTML & CSS, how about you create your own portfolio site. Think about the pages that you may include, such as home, about me and a portfolio or gallery page. You may even want to include a downloadable CV.
+
+## Further Reading:
+### Accessibility
+
+Some of your users might be using screenreaders or keyboards to navigate across your page.
+How do you ensure your website is available to them? By following a few guidelines and integrating techniques.
+
+#### A few names acronyms to know:
```

Comment:
How about changing this heading to just say,
> Acronyms to know

Also, I would include the `a11y` numeronym in this section too!

Reply:
Good Point!